JPS_Ontology/ontology/ontobms/OntoBMS.csv

```diff
@@ -46,3 +46,6 @@ conveys,Object Property,,,https://saref.etsi.org/saref4bldg/FlowMovingDevice,htt
 hasLowLevelState,Object Property,,,https://saref.etsi.org/core/MultiLevelState,LowLevelState,,A relationship that describes a low level state of a multi-level state.,https://www.theworldavatar.com/kg/ontobms/,has low level state
 hasMediumLevelState,Object Property,,,https://saref.etsi.org/core/MultiLevelState,MediumLevelState,,A relationship that describes a medium level state of a multi-level state.,https://www.theworldavatar.com/kg/ontobms/,has medium level state
 hasHighLevelState,Object Property,,,https://saref.etsi.org/core/MultiLevelState,HighLevelState,,A relationship that describes a high level state of a multi-level state.,https://www.theworldavatar.com/kg/ontobms/,has high level state
+suppliesAirTo,Object Property,,,https://saref.etsi.org/core/HVAC,https://saref.etsi.org/saref4bldg/FlowMovingDevice,,A relationship that describes a HVAC device supplying air to a flow moving device.,https://www.theworldavatar.com/kg/ontobms/,supplies air to
+extractsAirFrom,Object Property,,,https://saref.etsi.org/core/HVAC,https://saref.etsi.org/saref4bldg/FlowMovingDevice,,A relationship that describes a HVAC device removing air from a flow moving device or a flow treatment device.,https://www.theworldavatar.com/kg/ontobms/,extracts air from
+extractsAirFrom,Object Property,,,https://saref.etsi.org/core/HVAC,https://saref.etsi.org/saref4bldg/FlowTreatmentDevice,,A relationship that describes a HVAC device removing air from a flow moving device or a flow treatment device.,https://www.theworldavatar.com/kg/ontobms/,extracts air from
```

Comment:
You will need to define this object property on a single line in your csv file using the UNION keyword for the range. Otherwise you end up with two separate range definitions which is interpreted as _intersection_ (which would be wrong in this case).

Reply:
Oh okay, I will modify that. Thank you